### DISCUSSION TITLE
More beautiful and modern icons for EOS (like that of Catalina for reference)

### DISCUSSION THREAD

**Discussion by janardannn** [elementary/icons#1023] [Ideas]
Hey, I think EOS needs some more modern looking icons, no offence to the current ones but they look really old. And new icons would look much better with the dark theme and new fonts I guess.

**Comment by cassidyjames** [comment]
1. ~This discussion board is for suggestions of things to discuss on livestreams. This is not a catch-all for ideas for anything elementary-related; I'm moving this to the elementary/icons repo.~ It has been transferred.

2. The elementary icons have been iterated on for a decade with significant updates over that time. "They look really old" makes no sense and is not any sort of actionable information.

3. "New icons" is not just something someone does in one sweep; we solve actual raised issues by iterating. We value consistency and don't just redesign things for the sake of redesigning.

4. macOS Catalina has its own visual style that is pretty far away from that of elementary OS; we don't have any desire to copy the macOS icon style.
